deployer/src/proxy.rs

```diff
@@ -81,6 +81,8 @@ pub async fn handle(
         Ok(Some(address)) => address,
         Ok(None) => {
             trace!(?host, service, "service not found on this server");
+            Span::current().record("http.status_code", StatusCode::NOT_FOUND.as_u16());
```

Comment:
I'm thinking the proxy should return a unique status code so that we know something went wrong in the proxy (and not the users app)

Reply:
Yeah, that makes sense!

Reply:
I added a new field `proxy.status_code` for the status code from the proxy itself.